qcengine/programs/tests/test_ccsd.py

```diff
@@ -0,0 +1,77 @@
+import copy
```

Comment:
Maybe `test_standard_suite_correlated.py` or similar? I think we will end up building a lot of these standard test suites.

Reply:
this is only 1/6 for ccsd since there's full/fc, r/u/ro. also more `InputError` tests for what the programs can't do. so going with `test_standard_suite_ccsd.py`